lib/sender.js

```diff
@@ -81,7 +81,7 @@ class Sender {
 
     if (!options.mask) return [target, data];
 
-    randomFillSync(mask, 0, 4);
```

Comment:
Ok this explains a lot so no need to overdo! Forget about my previous ideas with  ring buffers :)